src/app/loadout/ModPicker.tsx

```diff
@@ -154,6 +159,12 @@ function mapStateToProps() {
               maxSelectable,
               plugs,
             };
+            if (
+              maxSelectable === 1 &&
+              armor2PlugCategoryHashes.includes(plugs[0].plug.plugCategoryHash)
+            ) {
```

Comment:
I wonder if we should just add the `plugSetHash` to `PlugsWithMaxSelectable` and use that? 

I kinda want to turn it into `PlugSetWithMaxSelectable` so we can check the `plugSetHash` as needed. 

Happy to leave it out if you would rather this, but just a thought.

Reply:
yeah i am not against waiting to see if something changes, the stakes are pretty low. if you do it with d2ai it's probably wrong at the very beginning of every season